### PR TITLE
ci: Use vars.EDITION instead of hard-coded edition to avoid conflicts with EE sync

### PR DESCRIPTION
**Pull request by sharat87**
Note: Please review/approve only, do not merge. This PR is expected to cause conflicts and so I'd like to be around when this is merged.

### CODE REVIEW

File: .github/workflows/fat-migration.yml
Comment (sharat87) [inline .github/workflows/fat-migration.yml:1]: @Aishwarya-U-R, are we still using this workflow file today? If not, can we delete it please?

Comment (Aishwarya-U-R) [reply .github/workflows/fat-migration.yml:1]: Not used 🤔

Comment (sharat87) [reply .github/workflows/fat-migration.yml:1]: Deleting the file then. Thanks for confirming.

Comment (Aishwarya-U-R) [reply .github/workflows/fat-migration.yml:1]: Have deleted it & PR is merged to release @sharat87 🙏🏻

**Review by sarojsarab — APPROVED** [review]
LGTM

**Pull request merged**